src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf

```diff
@@ -426,6 +426,10 @@
                 <source>Using hidden overlay characters is not allowed.</source>
                 <target>Draudžiama naudoti paslėptus perdangos simbolius.</target>
             </trans-unit>
+            <trans-unit id="110">
+                <source>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</source>
+                <target>"Failo plėtinys netinkamas ({{ extension }}). Leidžiami plėtiniai yra {{ extensions }}."</target>
```

Comment:
whole translation shouldn't be in quotes

Reply:
Of course, thanks!